security/src/main/java/zone/cogni/lib/security/common/SecurityConfiguration.java

```diff
@@ -30,16 +31,22 @@ private void init() {
 
   @Configuration
   @ConditionalOnProperty(name = "cognizone.security.auth-method", havingValue = "basic")
-  @EnableBasicAuth
+  @EnableSecurityBasicAuth
   public static class GenericBasicAuthConfiguration {
   }
 
   @Configuration
   @ConditionalOnProperty(name = "cognizone.security.auth-method", havingValue = "saml2")
-  @EnableSaml2
+  @EnableSecuritySaml2
   public static class GenericSaml2Configuration {
   }
 
+  @Configuration
+  @ConditionalOnProperty(name = "cognizone.security.auth-method", havingValue = "disabled")
```

Comment:
should the value `disabled` and the `@EnableSecurityOff` not be aligned? well... that only makes sense if you have both exposed 🤔

Reply:
yes, will rename the property to `off` (@EnableSecurityDisabled is too weird)

Reply:
true